### PR TITLE
Minor tweaks for the registration form article

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | - |

### CODE REVIEW

**Comment by OskarStark** [comment]
:+1:

File: cookbook/doctrine/registration_form.rst [hunk 1]
```diff
@@ -19,12 +19,12 @@ first start with :doc:`/cookbook/security/entity_provider`.
 
 Your ``User`` entity will probably at least have the following fields:
 
-``username``
+* ``username``
```
Comment (WouterJ) [inline cookbook/doctrine/registration_form.rst]: can you please explain why you did this? The previous definition list syntax seems better than the unordered list syntax.

Comment (javiereguiluz) [inline cookbook/doctrine/registration_form.rst]: If you browse this page (http://symfony.com/doc/current/cookbook/doctrine/registration_form.html) you'll understand why I did this :)

symfony_doc_error

Comment (WouterJ) [inline cookbook/doctrine/registration_form.rst]: Isn't it better to change the other 2 to be a definition list?

**Comment by OskarStark** [comment]
:+1:  for definition instead of an unordered list

File: cookbook/doctrine/registration_form.rst (hunk 1 above)
Comment (javiereguiluz) [inline cookbook/doctrine/registration_form.rst]: Yes. Done. Thanks.

**Comment by WouterJ** [comment]
:+1:

**Comment by xabbuh** [comment]
:+1:

**Comment by xabbuh** [comment]
Thanks Javier.

**Pull request closed without merging**